scrapy/interfaces.py

```diff
@@ -15,3 +15,22 @@ def list():
 
     def find_by_request(request):
         """Return the list of spiders names that can handle the given request"""
+
+
+class IAddon(Interface):
+    """Scrapy add-on"""
+
+    name = Attribute("""Add-on name""")
+    version = Attribute("""Add-on version string (PEP440)""")
```

Comment:
As these attributes don't seem to be required, I'm not sure we need this interface.